OWNERS.md

```diff
@@ -1,16 +1,32 @@
-# OWNERS
+# Crossplane Maintainers
 
-This page lists all maintainers for **this** repository. Each repository in the [Crossplane
-organization](https://github.com/crossplane/) will list their repository maintainers in their own
-`OWNERS.md` file.
+This page lists all active maintainers and reviewers for **this** repository.
+Each repository in the [Crossplane organization](https://github.com/crossplane/)
+will list their repository maintainers and reviewers in their own `OWNERS.md`
+file.
+
+Please see [GOVERNANCE.md](https://github.com/crossplane/crossplane/blob/masterGOVERNANCE.md)
+for governance guidelines and responsibilities for maintainers, and reviewers.
+
+See [CODEOWNERS](CODEOWNERS) for automatic PR assignment.
 
-Please see the Crossplane
-[GOVERNANCE.md](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md) for governance
-guidelines and responsibilities for the steering committee and maintainers.
 
 ## Maintainers
 
 * Nic Cope <negz@upbound.io> ([negz](https://github.com/negz))
 * Daniel Mangum <dan@upbound.io> ([hasheddan](https://github.com/hasheddan))
-* Muvaffak Onuş <monus@upbound.io> ([muvaf](https://github.com/muvaf))
-* Hasan Türken <hasan@upbound.io> ([turkenh](https://github.com/turkenh))
+* Muvaffak Onus <monus@upbound.io> ([muvaf](https://github.com/muvaf))
+* Hasan Turken <hasan@upbound.io> ([turkenh](https://github.com/turkenh))
```

Comment:
Huh, interesting. I guess the file in this repo had correct characters but the file I copied from c/c did not. 🤔